### PR TITLE
Schema for projects and organizations

### PR DESCRIPTION
**Pull request by Ciszkoo**

### CODE REVIEW

File: src/server/db/schema.ts [hunk 1]
```diff
@@ -114,16 +119,73 @@ export const videoEntries = mysqlTable("videoEntry", {
...
+    }),
+  }),
+);
+
+export const projects = mysqlTable("project", {
+  id: bigint("id", { mode: "number" }).primaryKey().autoincrement(),
+  projectName: varchar("projectName", { length: 256 }).notNull(),
+  projectDescription: varchar("projectDescription", { length: 512 }),
+  title: varchar("title", { length: 256 }),
+  description: varchar("description", { length: 512 }),
```
Comment (hubcio2115) [inline src/server/db/schema.ts:177]: What's the difference between `projectDescription` and `description`?

**Review by hubcio2115 — CHANGES_REQUESTED** [review]

File: src/server/db/schema.ts (hunk 1 above)
Comment (Ciszkoo) [reply src/server/db/schema.ts:177]: In `projectDescription` users in organization can put whatever they want, `description` is a part of YT metadata. Can be merged into one, idc

Comment (hubcio2115) [reply src/server/db/schema.ts:177]: Leave it

**Review by hubcio2115 — APPROVED** [review]
LGTM!

**Review by hubcio2115 — APPROVED** [review]
LGTM! 👍

**Pull request merged**